modules/Books.js

```diff
@@ -0,0 +1,54 @@
+export default class Books {
+  constructor() {
+    this.booksList = [];
+  }
+
+  #SetBooksList = (books) => {
+    localStorage.setItem('Books', JSON.stringify(books));
+  };
+
+  #getBooksList = () => {
+    const books = JSON.parse(localStorage.getItem('Books'));
+    if (books) {
+      return books;
+    }
+    return [];
+  };
+
+  newCollection = (bookValues) => {
+    this.booksList = this.#getBooksList();
+    this.booksList.push(bookValues);
+    this.#SetBooksList(this.booksList);
+    document.querySelector('#add-new-book').style.display = 'none';
+    document.querySelector('#show-books').style.display = 'flex';
+  };
+
+  deleteCollection = (bookIndex) => {
+    let bookRemoved;
+    if (bookIndex !== null) {
+      const books = this.#getBooksList();
+      bookRemoved = books.filter((_item, key) => {
+        if (key === parseInt(bookIndex, 10)) {
+          return null;
+        }
+        return true;
+      });
+      this.#SetBooksList(bookRemoved);
+      this.showBooks();
+    }
+  };
```

Comment:
- [ ] Good job so far, however, I was not able to remove a book when I clicked the `remove` button. Kindly fix this so that your project works as expected.